BuildPropertiesPlugin/sample/app/build.gradle

```diff
@@ -1,15 +1,11 @@
 buildscript {
-  repositories {
-    maven { url rootProject.file('../.gradle/repo') }
-  }
   dependencies {
     classpath 'com.android.tools.build:gradle:2.1.2'
-    classpath "${pluginGroup}:${pluginId}:${pluginVersion}"
   }
 }
 
 apply plugin: 'com.android.application'
-apply plugin: 'build-properties'
+apply plugin: com.novoda.buildproperties.BuildPropertiesPlugin
```

Comment:
This is long :) 
I think just `com.novoda.buildproperties` would be cool. But I guess this didn't change in this PR at all, right?